build/build.proj

```diff
@@ -21,7 +23,7 @@
   </Target>
 
   <Target Name="Build" DependsOnTargets="RestorePackages">
-    <Exec Command="$(MSBuild) ..\src\openstack.net.sln /p:Configuration=$(Configuration) /nologo /v:minimal" WorkingDirectory="$(MSBuildThisFileDirectory)" />
+    <Exec Command="$(MSBuild) ..\src\openstack.net.sln /p:Configuration=$(Configuration) /p:Platform=$(Platform) /nologo /v:minimal" WorkingDirectory="$(MSBuildThisFileDirectory)" />
```

Comment:
:memo: If you change the `<Platform>` property to remove the quotes as described previously, this line will need to quote the associated command line argument.